apps/condo/domains/organization/schema/Organization.js

```diff
@@ -36,6 +37,20 @@ const Organization = new GQLListSchema('Organization', {
             isRequired: true,
             kmigratorOptions: { null: false },
         },
+        type: {
+            schemaDoc: 'Type of organization',
+            type: Select,
+            options: ORGANIZATION_TYPES,
+            dataType: 'string',
+            defaultValue: MANAGING_COMPANY_TYPE,
```

Comment:
What about all existing organizations? Maybe provide special script that could manage it's type relying on `OrganizationLink` connection?

Reply:
See migration

Reply:
Yup, that's good